src/components/UserSearch/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { getRepos, getUserInfo } from '../../redux/actions';
+import github from '../../images/github.svg';
+
+export default () => {
+	const [input, setInput] = useState('');
+	const dispatch = useDispatch();
+
+	const handleChange = (e) => setInput(e.target.value);
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		dispatch(getRepos(input));
+		dispatch(getUserInfo(input));
+		setInput('');
+	};
+
+	return (
+		<>
+			<div className="search-element d-flex justify-content-center sticky-top">
+				<img className="logo" src={github} alt="Github logo" />
```

Comment:
running your site through `npm run dev`, the GitHub logo is showing as an img alt. 
I just got an email to say Parth just tried to change this so I pulled your changes and reran it and the new file for the GitHub logo arrived, however it still shows up like this: 
![Screenshot 2021-09-15 at 14 29 43](https://user-images.githubusercontent.com/60107486/133442647-6dac727d-80c0-41e2-9dcb-cd83c20be104.png)

![Screenshot 2021-09-15 at 14 32 08](https://user-images.githubusercontent.com/60107486/133442750-864419f8-c00d-4b2c-a0c4-2878b18213fe.png)